### ISSUE TITLE
Multipass description does not show the alias name

### ISSUE THREAD

**Issue by jamiedbennett** [CanonicalLtd/multipass#11]
The command to run multipass is `multipass`, not `multipass.ubuntu`.

```
jamie@ubik:~$ snap info multipass
name:      multipass
summary:   Ubuntu at your fingertips
publisher: saviq
description: |
  Multipass gives you Ubuntu VMs in seconds. Just run `multipass.ubuntu create`
  and
  it'll do all the setup for you.
snap-id: mA11087v6dR3IEcQLgICQVjuvhUUBUKM
commands:
  - multipass
services:
  multipass.multipassd: simple, enabled, active
tracking:               edge
installed:              2017.2.1-3-g248ab13 (31) 44MB devmode
refreshed:              2017-12-07 11:45:27 +0000 UTC
channels:                                        
  stable:               –                             
  candidate:            –                             
  beta:                 2017.2.2            (37) 44MB classic
  edge:                 2017.2.2-4-g691449f (38) 44MB classic

```

**Comment by popey** [comment]
Dupe of bug #5

**Comment by Saviq** [comment]
Closed by #5
